gramps/gui/configure.py

```diff
@@ -1108,6 +1108,13 @@ def add_formats_panel(self, configdialog):
                           row, 'interface.grampletbar-close', stop=3,
                           extra_callback=self.cb_grampletbar_close)
         row += 1
+
+        # When using Select <item>, do the trees expand by default?
+        self.add_checkbox(grid,
+                          _("Expand trees by default."),
```

Comment:
Change to "Expand selectors by default".   The trailing full stop is not needed.